### PR TITLE
save master on Nelson machine (2/3)

### PR DESCRIPTION
**Pull request by amigaire**
This is the code version on the Nelson docker machine.
We take the opportunity to update the master branch with these code version.

1/3: https://github.com/akeneo/nelson/pull/73
**2/3: https://github.com/akeneo/nelson/pull/72**
3/3: https://github.com/akeneo/nelson/pull/71

### CODE REVIEW

File: src/Akeneo/Git/PullRequestMerger.php [hunk 1]
```diff
@@ -21,10 +21,11 @@ public function mergePullRequest(array $pullRequest): void
 
         $mergeTitle = sprintf('Merge pull request #%s', $pullRequest['number']);
 
-        $this->eventDispatcher->dispatch(
+	$this->eventDispatcher->dispatch(
```
Comment (amigaire) [inline src/Akeneo/Git/PullRequestMerger.php:24]:
```suggestion
        $this->eventDispatcher->dispatch(
```

File: src/Akeneo/Git/PullRequestMerger.php [hunk 2]
```diff
@@ -33,10 +34,10 @@ public function mergePullRequest(array $pullRequest): void
             $pullRequest['base']['repo']['name'],
             $pullRequest['number'],
             $mergeTitle,
-            $pullRequest['base']['sha'],
+            $pullRequest['head']['sha'],
         );
 
-        $this->eventDispatcher->dispatch(
+	$this->eventDispatcher->dispatch(
```
Comment (amigaire) [inline src/Akeneo/Git/PullRequestMerger.php:40]:
```suggestion
        $this->eventDispatcher->dispatch(
```

File: src/Akeneo/Git/PullRequestMerger.php [hunk 3]
```diff
@@ -21,10 +21,11 @@ public function mergePullRequest(array $pullRequest): void
 
         $mergeTitle = sprintf('Merge pull request #%s', $pullRequest['number']);
 
-        $this->eventDispatcher->dispatch(
+	$this->eventDispatcher->dispatch(
             new GenericEvent($this, [
                 'number' => $pullRequest['number'],
-            ]),
+	        'commit_sha' => $pullRequest['head']['sha'],// TODO remove
+	    ]),
```
Comment (amigaire) [inline src/Akeneo/Git/PullRequestMerger.php:28]: Fix to be done in another PR
```suggestion
```

File: src/Akeneo/Git/PullRequestMerger.php [hunk 4]
```diff
@@ -33,10 +34,10 @@ public function mergePullRequest(array $pullRequest): void
             $pullRequest['base']['repo']['name'],
             $pullRequest['number'],
             $mergeTitle,
-            $pullRequest['base']['sha'],
+            $pullRequest['head']['sha'],
```
Comment (amigaire) [inline src/Akeneo/Git/PullRequestMerger.php:37]: Fix to be done in another PR

```suggestion
```

**Review by pchasle — APPROVED** [review]

**Review by nmarniesse — APPROVED** [review]

**Review by StevenVAIDIE — APPROVED** [review]

**Review by LevFlavien — APPROVED** [review]

**Review by Beninho — APPROVED** [review]
